src/SFA.DAS.AssessorService.Api.Types/Models/Certificates/GetApprovedDraftCertificatesRequest.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+using MediatR;
+
+namespace SFA.DAS.AssessorService.Api.Types.Models.Certificates
+{
+    public class GetApprovedDraftCertificatesRequest : IRequest<List<CertificateSummaryResponse>>
+    {     
```

Comment:
Is this empty by design? If so can you pop a quick comment in here.

Reply:
done